runtime/tutor.txt

```diff
@@ -303,29 +309,29 @@ _________________________________________________________________
 
 
 
-
-
-
 =================================================================
 =                             RECAP                             =
 =================================================================
 
- * Type w to select forward until the next word.
-   * Type e to select to the end of the current word.
-   * Type b to select backward to the start of the current word.
+ * Press w to select forward until the next word.
+   * Press e to select to the end of the current word.
+   * Press b to select backward to the start of the current word.
    * Use uppercase counterparts, W,E,B, to traverse WORDS.
 
  * Typing d deletes the entire selection, so you can delete a
    word forward by typing wd.
 
- * Type c to delete the selection and enter Insert mode.
+ * Press c to delete the selection and enter Insert mode.
 
- * Type a number before a motion to repeat it that many times.
+ * Press a number before a motion to repeat it that many times.
 
- * Type x to select the entire current line. Type x again to
+ * Press x to select the entire current line. Press x again to
    select the next line.
 
- * Type u to undo. Type U to redo.
+ * Press u to undo. Press U to redo.
```

Comment:
Using "type" is consistent with vimtutor, and because of the technicality that commands such as `U` are not a single key*press*.

Reply:
Some places say type even when it's just one keypress while others just say press, for the sake of consistency, should I convert the one-keypresses to say press, and others to say type, or just convert everything to say type?

Reply:
I've gone through and started changing some "press"s to "type"s and vice versa. Although it's quite hard to stick to a clear convention because of a two main reasons.
1. It could confuse / irritate the user slightly by alternating between type and press so often
2. Sometimes when you're referring to to mappings like f/F one is a "press" and one is a "type" if you count the shift key, which you say you do.

What I've done in case 2 is just say "use" so technically there is a convention:
- "Press" for single keypresses
- "Type" for multiple keypresses / modifiers
- "Use" when referencing two inputs as a pair.

So yeah, technically resolved unless you have any objections. The switching between wordings may prove jarring and I can fix that with any complaints/alternative solutions.